app-ios/Sources/Theme/Resources/Colors.xcassets/black.colorset/Contents.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Can you remove this file?
This is not needed for current situation 🙏🏼

Reply:
I deleted at https://github.com/DroidKaigi/conference-app-2022/pull/656/commits/bed0feda90979219bb32e1ab140bf9a9dc05aec6 !